src/MovieCard/MovieCard.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
```

Comment:
Think we can get rid of this since we dont have it in the other functional component files